Mage.Client/pom.xml

```diff
@@ -127,11 +127,6 @@
             <artifactId>jlayer</artifactId>
             <version>1.0.1</version>
         </dependency>
-        <dependency>
-            <groupId>org.mobicents.external.tritonus</groupId>
```

Comment:
What did this dependancy do?

Reply:
Good question! If I recall correctly, it was an audio library the xmage no longer uses.

Reply:
Can you please look into that. Not a good idea to remove dependancies if we're not sure what they're for. If it's no longer used can you point to when it got refactored out.